wrapper/src/main/java/software/amazon/jdbc/exceptions/MultiAzDbClusterPgExceptionHandler.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.exceptions;
+
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class MultiAzDbClusterPgExceptionHandler implements ExceptionHandler {
```

Comment:
What's the differences comparing to PgExceptionHandler? Does it make sense to use an existing handler?

Reply:
`28000` is the only change. we should refactor it as a whole.

Reply:
refactored with latest PR.